controllers/cinder_controller.go

```diff
@@ -67,9 +67,9 @@ func (r *CinderReconciler) GetKClient() kubernetes.Interface {
 	return r.Kclient
 }
 
-// GetLogger -
-func (r *CinderReconciler) GetLogger() logr.Logger {
-	return r.Log
+// getLog returns a logger object with a logging prefix of "conrollers.name" and aditional controller context fields
+func cinderLog(ctx context.Context) logr.Logger {
```

Comment:
-1: Please make this a struct method like it was before, all you need to do is add the new method argument.

```
func (r *CinderReconciler) GetLogger(ctx context.Context) logr.Logger {
```

That way the name remains the same regardless of the controller we are in and we'll always do `log := r.GetLogger(ctx)`